examples/miscellaneous/plot_pipeline_display.py

```diff
@@ -0,0 +1,215 @@
+"""
+=================================================================
+Displaying Pipelines
+=================================================================
+
+The default configuration for displaying a pipeline is `'text'` where
+`set_config(display='text')`.  To visualize the diagram in Jupyter Notebook,
+use `set_config(display='diagram')` and then output the pipeline object.
+
+To see more detailed steps in the visualization of the pipeline, click on the
+steps in the pipeline.
+"""
+
+# %%
+# Displaying a Basic Pipeline with a Classifier
```

Comment:
I think we can remove the basic pipeline with one step because it is not a common use case.

(Also we usually want to scale before passing data into `LogisticRegression`.